plugin-server/tests/main/process-event.test.ts

```diff
@@ -1032,6 +1032,62 @@ test('snapshot event stored as session_recording_event', async () => {
     expect(event.snapshot_data).toEqual({ timestamp: 123 })
 })
 
+test('performance event stored as performance_event', async () => {
+    await eventsProcessor.processEvent(
+        'some-id',
+        '',
+        {
+            event: '$perfomance_event',
+            properties: {
+                // Taken a real event from the JS
+                '0': 'resource',
+                '1': 1671723295836,
+                '2': 'http://localhost:8000/api/projects/1/session_recordings',
+                '3': 10737.89999999106,
+                '4': 0,
+                '5': 0,
+                '6': 0,
+                '7': 10737.89999999106,
+                '8': 10737.89999999106,
+                '9': 10737.89999999106,
+                '10': 10737.89999999106,
+                '11': 0,
+                '12': 10737.89999999106,
+                '13': 10745.09999999404,
+                '14': 11121.70000000298,
+                '15': 11122.20000000298,
+                '16': 73374,
+                '17': 1767,
+                '18': 'fetch',
+                '19': 'http/1.1',
+                '20': 'non-blocking',
+                '22': 2067,
+                '39': 384.30000001192093,
+                '40': 1671723306573,
+                token: 'phc_234',
+                $session_id: '1853a793ad26c1-0eea05631cbeff-17525635-384000-1853a793ad31dd2',
+                $window_id: '1853a793ad424a5-017f7473b057f1-17525635-384000-1853a793ad524dc',
+                distinct_id: '5AzhubH8uMghFHxXq0phfs14JOjH6SA2Ftr1dzXj7U4',
+                $current_url: 'http://localhost:8000/recordings/recent',
+            },
+        } as any as PluginEvent,
+        team.id,
+        now,
+        new UUIDT().toString()
+    )
+    await delayUntilEventIngested(() => hub.db.fetchPerformanceEvents())
+
+    const events = await hub.db.fetchEvents()
+    expect(events.length).toEqual(0)
+
+    const sessionRecordingEvents = await hub.db.fetchPerformanceEvents()
+    expect(sessionRecordingEvents.length).toBe(1)
+
+    const [event] = sessionRecordingEvents
+    console.log(event)
+    expect(event).toEqual({})
```

Comment:
assuming unfinished 🤣

Reply:
Yeah I can't get the tests to run locally so I'm gonna see what the CI says :D